docs/admin/install/helm-install/google-gke.md

```diff
@@ -3,6 +3,210 @@ tags:
+| config-api        | 0.5      | 0.5GB | N/A        | 64 Bit         | No                                 |
+
+Releases of images are in style 1.0.0-beta.0, 1.0.0-0
+
+## Initial Setup
+
+1.  If you are using Cloud Shell, you can skip to step 4.
+
+2.  Install [gcloud](https://cloud.google.com/sdk/docs/quickstarts)
+    
+3.  Install kubectl using `gcloud components install kubectl` command
+    
+4.  Create cluster using a command such as the following example:
+
+    ```  
+    gcloud container clusters create jans-cluster --num-nodes 2 --machine-type e2-highcpu-8 --zone us-west1-a
+    ```
+    You can adjust `num-nodes` and `machine-type` as per your desired cluster size
+
+5.  Install [Helm3](https://helm.sh/docs/intro/install/)    
+
+6.  Create `jans` namespace where our resources will reside
+    ```
+    kubectl create namespace jans
+    ```
+
+## Jans Installation using Helm
+1.  Install [Nginx-Ingress](https://github.com/kubernetes/ingress-nginx), if you are not using Istio ingress
+    
+      ```
+      helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx
+      helm repo add stable https://charts.helm.sh/stable
+      helm repo update
+      helm install nginx ingress-nginx/ingress-nginx
+      ```
+
+2.  Create a file named `override.yaml` and add changes as per your desired configuration:
+
+    - FQDN/domain *not* registered:
+    
+        
+        Get the Loadbalancer IP: 
+        ```
+        kubectl get svc nginx-ingress-nginx-controller --output jsonpath='{.status.loadBalancer.ingress[0].ip}'
+        ```
+
+      
+        
+        Add the following yaml snippet to your `override.yaml` file:
+
+        ```yaml
+        global:
+            lbIp: #Add the Loadbalance IP from the previous command
+            isFqdnRegistered: false
+        ```
+
+    - FQDN/domain registered:
+
+        Add the following yaml snippet to your `override.yaml` file`:
+
+        ```yaml
+        global:
+            lbIp: #Add the LoadBalancer IP from the previous command
+            isFqdnRegistered: true
+            fqdn: demoexample.jans.org #CHANGE-THIS to the FQDN used for Jans
+        nginx-ingress:
+          ingress:
+              path: /
+              hosts:
+              - demoexample.jans.org #CHANGE-THIS to the FQDN used for Jans
+              tls:
+              - secretName: tls-certificate
+                hosts:
+                - demoexample.jans.org #CHANGE-THIS to the FQDN used for Jans
+        ```
+
+
+
+
+
+
+    -  LDAP/Opendj for persistence storage
+
+
+          Add the following yaml snippet to your `override.yaml` file:
+          ```yaml
+          global:
+            cnPersistenceType: ldap
+            storageClass:
+              provisioner: kubernetes.io/gce-pd
+            opendj:
+              enabled: true
+          ```
+
+          So if your desired configuration has no-FQDN and LDAP, the final `override.yaml` file will look something like that:
```

Comment:
where id the nginx-ingress section go that you asked the user to add above?

Reply:
I am covering no-FQDN, so it there's no nginx-ingress section. only LBaddr is needed.
In the other override.yaml example, which covers mysql and registered fqdn, it has the nginx-ingress section.

Reply:
nginx ingress is needed in both cases . I'm not following

Reply:
```
       nginx:
          ingress:
              enabled: true
              path: /
              hosts:
              - demoexample.jans.org #CHANGE-THIS to the FQDN used for Jans
              tls:
              - secretName: tls-certificate
                hosts:
                - demoexample.jans.org #CHANGE-THIS to the FQDN used for Jans
                ```